README.md

```diff
@@ -18,31 +18,23 @@ The `{tlf}` package provides an object-oriented framework to create
 tables and figures, which are used by R packages in the Open Systems
 Pharmacology ecosystem:
 
--   [`{ospsuite}`](https://www.open-systems-pharmacology.org/OSPSuite-R/)
--   [`{ospsuite.reportingengine}`](https://www.open-systems-pharmacology.org/OSPSuite.ReportingEngine/)
+- [`{ospsuite}`](https://www.open-systems-pharmacology.org/OSPSuite-R/)
+- [`{ospsuite.reportingengine}`](https://www.open-systems-pharmacology.org/OSPSuite.ReportingEngine/)
 
 ## Installation
 
-You can install the development version of `{tlf}` from
-[GitHub](https://github.com/) with:
-
+You can install the latest release of `{tlf}` directly from Github with:
 ``` r
 # install.packages("remotes")
-remotes::install_github("Open-Systems-Pharmacology/TLF-Library")
+remotes::install_github("Open-Systems-Pharmacology/TLF-Library@*release")

```

Comment:
Because together with this line in DESCRIPTION
```
Remotes: github::Open-Systems-Pharmacology/OSPSuite.RUtils@*release
```

It will install the packages and its **releases** dependencies in just one line of code. Easier, no ?